test/Microsoft.AspNetCore.Razor.Language.Test/TestFiles/IntegrationTests/CodeGenerationIntegrationTest/DesignTime_DesignTime.codegen.cs

```diff
@@ -49,13 +49,14 @@ public class TestFiles_IntegrationTests_CodeGenerationIntegrationTest_DesignTime
 
 #line default
 #line hidden
-            DefineSection("Footer", async (__razor_section_writer) => {
+            DefineSection("Footer", async(__razor_section_writer) => {
 #line 14 "TestFiles/IntegrationTests/CodeGenerationIntegrationTest/DesignTime.cshtml"
 __o = bar;
 
 #line default
 #line hidden
-            });
+            }
+            );
```

Comment:
Offhand remark.

There's just sooooo much policy in some of these csharpcodewriter methods. We should think about making them internal or deleting them.

The fact that I can't use `BuildAsyncLambda` without the newline here means that I really don't want to use it.